src/Infrastructure/FileSyncer/PhpFileSyncer.php

```diff
@@ -51,90 +66,59 @@ public function sync(
         ?ProcessOutputCallbackInterface $callback = null,
         ?int $timeout = 120
     ): void {
-        $source = DirectoryUtil::stripTrailingSlash($source);
-        $destination = DirectoryUtil::stripTrailingSlash($destination);
+        $this->source = $source;
```

Comment:
It's interesting that these should be passed in the constructor. I would have assumed that they should be passed at call time. What's the rationale for this decision?

Reply:
This isn't actually the constructor; it just looks like it because of where GitHub collapsed the code above. If you expand it, you'll see that this is the `::sync()` method, like you would expect. 🙂